### PR TITLE
Use correct label when querying pod and fix image imports

### PR DESCRIPTION
**Pull request by michaelsauter**
src/org/ods/usecase/LeVADocumentUseCase.groovy#L1024 must have been a mistake when resolving merge conflicts - the return value is definitely not used.

### CODE REVIEW

**Review by metmajer — APPROVED** [review]

File: src/org/ods/service/OpenShiftService.groovy [hunk 1]
```diff
@@ -253,9 +253,9 @@ class OpenShiftService {
       ''
     }
 
-    Map getPodDataForComponent(String name) {
+    Map getPodDataForComponent(String componentSelector) {
         String stdout = this.steps.sh(
-          script: "oc get pod -l component=${name} -o json --show-all=false",
+          script: "oc get pod -l ${componentSelector} -o json --show-all=false",
           returnStdout: true,
           label: "Getting OpenShift Pod data for ${name}"
```
Comment (clemensutschig) [inline src/org/ods/service/OpenShiftService.groovy]: wrong {} .. should be `componentSelector`

File: src/org/ods/usecase/LeVADocumentUseCase.groovy [hunk 2]
```diff
@@ -1021,8 +1021,6 @@ class LeVADocumentUseCase extends DocGenUseCase {
         }
         def sectionsNotDone = this.getSectionsNotDone(sections)
 
-        def pods = this.os.getPodDataForComponent(repo.id)
-
         def data_ = [
```
Comment (clemensutschig) [inline src/org/ods/usecase/LeVADocumentUseCase.groovy:1029]: you create in the `data` object now an openshift structure. that needs to be reflected in the lines  1030ff

**Comment by michaelsauter** [comment]
@metmajer @clemensutschig Fixed the issues, and also one I introduced yesterday with the image import.

**Review by clemensutschig — APPROVED** [review]
works! @metmajer  please merge to master

**Comment by metmajer** [comment]
Thanks @michaelsauter @clemensutschig!

**Pull request merged**